### PR TITLE
added template for PRs

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
PLEASE merge together with https://github.com/kiegroup/kie-jenkins-scripts/pull/803

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+[link](https://www.example.com)
+
+**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_
+
+paste the link from GitHub here
+
+**How to retest**:
+
+<pre>  
+* to retest a pull request please edit:             regex [.*[j|J]enkins,?.*(retest|test) this.*] OR ok to test
```
Comment (mareknovotny) [inline .github/pull_request_template.md:14]: i would replace `edit` with `add comment`

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  
+* to run a full downstream builds please edit:      regex [.*[j|J]enkins,?.*(execute|run|trigger|start|do) fdb.*]
```
Comment (mareknovotny) [inline .github/pull_request_template.md:16]: i would replace `edit` with `add comment`

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  
+* to run a compile downstream build please edit:    regex [.*[j|J]enkins,?.*(execute|run|trigger|start|do) cdb.*]
```
Comment (mareknovotny) [inline .github/pull_request_template.md:18]: i would replace `edit` with `add comment`

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+* to run a full production downstream please edit:  regex [.*[j|J]enkins,?.*(execute|run|trigger|start|do) product fdb.*]
```
Comment (mareknovotny) [inline .github/pull_request_template.md:20]: i would replace `edit` with `add comment`

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+
+* to run an upstream build please edit:             regex [.*[j|J]enkins,?.*(execute|run|trigger|start|do) upstream.*]
```
Comment (mareknovotny) [inline .github/pull_request_template.md:22]: i would replace `edit` with `add comment`

**Comment by mbiarnes** [comment]
All changes done.
For merging - first https://github.com/kiegroup/kie-jenkins-scripts/pull/803 has to be merged. Once the seed-job was executed this PR can be merged.

**Comment by mareknovotny** [comment]
we should announce this on bsig ML too before it is merged @mbiarnes

**Comment by mbiarnes** [comment]
@mareknovotny will do this tomorrow

**Pull request merged**